e2e/startup-profiles.e2e.spec.ts

```diff
@@ -200,5 +221,63 @@ test.describe.serial('track startup windows based on existing profiles and setti
         expect(contents).toContain(`expecting value of type string, got an array ["strawberries","limes"]`);
       }
     });
+
+    test('missing version in defaults deployment profile', async() => {
+      const filename = `${ __filename }-missing-version-in-defaults-profile`;
+      const logDir = reportAsset(filename, 'log');
+      const logPath = path.join(logDir, 'background.log');
+
+      test.skip(skipReasons.length > 0, `Profile requirements for this test: ${ skipReasons.join(', ') }`);
+      if (process.platform === 'win32') {
+        await createDefaultUserRegistryProfileWithValidDataButNoVersion();
+      } else {
+        // Use JSON.parse to bypass the typescript type-checker
+        const s = `{"kubernetes":{"enabled":false}}`;
+        const s1 = JSON.parse(s) as RecursivePartial<Settings>;
```

Comment:
Same as above:
```suggestion
        const s1 = {kubernetes:{enabled:false}} as unknown as RecursivePartial<Settings>;
```
Actually, isn't it a valid `RecursivePartial<Settings>` already?
```suggestion
        const s1 = {kubernetes:{enabled:false}};
```

Reply:
copy/pasto